Automattic/pocket-casts-ios
# Search: fix an issue when showing episodes

When a user performs a search and taps on an unavailable episode we load and display it through `EpisodeLoadingController`.

In a recent ticket a user informs us that the swipe down to dismiss is not working in this part, this PR fixes it by changing how the `EpisodeDetailViewController` is presented.

## To test

1. Do a clean install
2. Go to discover
3. Search for any term
4. Tap on an episode
5. Swipe down
6. ✅ View should dismiss
7. Tap on another episode from another podcast
8. ✅ Ensure all buttons are working correctly

## Checklist
